pkgs/applications/networking/instant-messengers/telegram/tdesktop/default.nix

```diff
@@ -44,6 +47,7 @@ mkDerivation rec {
     qtbase qtimageformats gtk3 libsForQt5.libdbusmenu enchant2 lz4 xxHash
     dee ffmpeg openalSoft minizip libopus alsaLib libpulseaudio range-v3
     tl-expected hunspell
+    tg_owt tg_owt.src # Source is used for include directories.
```

Comment:
Huh, is not installing the includes like you do sufficient?

Reply:
@jtojnar
Third party libraries are in-tree and output `tg_owtTargets.cmake` reference these third party include directories, which default to be under  `/build/source`.
So I patched CMakeLists of tg_owt to build with store path `$src`, and it references absolute paths to `tg_owt.src` (store path). That's why it is needed here.

All cmake files of tg_owt are written for in-tree build and don't handle install and include directories for install at all.
This is just an easy but of course quite ugly way to make it work.

Reply:
I patched cmake files in another way and it's now fixed.